File: Database/Assembly_SRPG_JP/GuerrillaShopScheduleParam.cs
// Decompiled with JetBrains decompiler
// Type: SRPG.GuerrillaShopScheduleParam
// Assembly: Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
// MVID: 85BFDF7F-5712-4D45-9CD6-3465C703DFDF
// Assembly location: S:\Desktop\Assembly-CSharp.dll

namespace SRPG
{
  public class GuerrillaShopScheduleParam
  {
    public int id;
    public string begin_at;
    public string end_at;
    public int accum_ap;
    public string open_time;
    public string cool_time;
    public GuerrillaShopScheduleAdvent[] advent;

    public bool Deserialize(JSON_GuerrillaShopScheduleParam json)
    {
      this.id = json.id;
      this.begin_at = json.begin_at;
      this.end_at = json.end_at;
      this.accum_ap = json.accum_ap;
      this.open_time = json.open_time;
      this.cool_time = json.cool_time;
      if (json.advent != null)
      {
        GuerrillaShopScheduleAdvent[] shopScheduleAdventArray = new GuerrillaShopScheduleAdvent[json.advent.Length];
        for (int index = 0; index < json.advent.Length; ++index)
        {
          shopScheduleAdventArray[index] = new GuerrillaShopScheduleAdvent();
          shopScheduleAdventArray[index].id = json.advent[index].id;
          shopScheduleAdventArray[index].coef = json.advent[index].coef;
        }
      }
      return true;
    }
  }
}

File: Database_V2/ParamFunctions_untested/GuerrillaShopScheduleParam.py
def GuerrillaShopScheduleParam(json):
    this={}#GuerrillaShopScheduleParamjson)
    if 'id' in json:
        this['id'] = json['id']
    if 'begin_at' in json:
        this['begin_at'] = json['begin_at']
    if 'end_at' in json:
        this['end_at'] = json['end_at']
    if 'accum_ap' in json:
        this['accum_ap'] = json['accum_ap']
    if 'open_time' in json:
        this['open_time'] = json['open_time']
    if 'cool_time' in json:
        this['cool_time'] = json['cool_time']
    #if(json.advent!=null)
        #GuerrillaShopScheduleAdventshopScheduleAdventArray=newGuerrillaShopScheduleAdvent[json.advent.Length]
        #for(intindex=0index<json.advent.Length++index)
            #shopScheduleAdventArray=newGuerrillaShopScheduleAdvent()
            #shopScheduleAdventArray.id=json.advent.id
            #shopScheduleAdventArray.coef=json.advent.coef
    #returntrue
return this
